README.md

```diff
@@ -39,53 +43,61 @@ The directory structure of your new project looks like this:
 │
-├── notebooks          <- Jupyter notebooks. Naming convention is a number (for ordering),
-│                         the creator's initials, and a short `-` delimited description, e.g.
-│                         `1.0-jqp-initial-data-exploration`.
+├── notebooks                <- Jupyter notebooks. Naming convention is a number (for ordering),
+│                               the creator's initials, and a short `-` delimited description, e.g.
+│                               `1.0-jqp-initial-data-exploration`.
 │
-├── references         <- Data dictionaries, manuals, and all other explanatory materials.
+├── references               <- AQA plan, Assumptions log, data dictionaries, and all other explanatory materials
+│   ├── aqa_plan.md          <- AQA plan for the project
+│   └── assumptions_log.md   <- where to log key assumptions to data / models / analyses
 │
-├── reports            <- Generated analysis as HTML, PDF, LaTeX, etc.
-│   └── figures        <- Generated graphics and figures to be used in reporting
+├── reports                  <- Generated analysis as HTML, PDF, LaTeX, etc.
+│   └── figures              <- Generated graphics and figures to be used in reporting
 │
-├── requirements.txt   <- The requirements file for reproducing the analysis environment, e.g.
-│                         generated with `pip freeze > requirements.txt`
+├── requirements.txt         <- The requirements file for reproducing the analysis environment, e.g.
+│                               generated with `pip freeze > requirements.txt`
 │
-├── src                <- Source code for use in this project.
-│   ├── __init__.py    <- Makes src a Python module
-│   │
-│   ├── data           <- Scripts to download or generate data
-│   │   └── make_dataset.py
-│   │
-│   ├── features       <- Scripts to turn raw data into features for modeling
-│   │   └── build_features.py
-│   │
-│   ├── models         <- Scripts to train models and then use trained models to make
-│   │   │                 predictions
-│   │   ├── predict_model.py
-│   │   └── train_model.py
-│   │
-│   └── visualization  <- Scripts to create exploratory and results oriented visualizations
-│       └── visualize.py
+├── setup.py                 <- makes project pip installable (pip install -e .) so src can be imported
 │
-└── tox.ini            <- tox file with settings for running tox; see tox.testrun.org
-```
+├── src                      <- Source code for use in this project.
+    ├── __init__.py          <- Makes src a Python module
+    │
+    ├── make_data            <- Scripts to download or generate data
+    │
+    ├── make_features        <- Scripts to turn raw data into features for modeling
+    │
+    ├── make_models          <- Scripts to train models and then use trained models to make predictions
+    │
+    ├── make_viz             <- Scripts to create exploratory and results oriented visualizations
```

Comment:
Pedantic change - could we change this to `make_visualisations`? Same in all other places.

Reply:
Done.